Formula/q/qmmp.rb

```diff
@@ -102,6 +92,9 @@ def install
       cmake_args << "-DCMAKE_MODULE_LINKER_FLAGS=-Wl,-undefined,dynamic_lookup"
     end
 
+    # Fix to recognize x11
+    inreplace "src/plugins/Ui/skinned/CMakeLists.txt", "PkgConfig::X11", "${X11_LDFLAGS}"
```

Comment:
We should upstream this first before using it.

Reply:
That is right. But is there anyone can find a way to set this by argument or else? Since this is really intended change and I am not c++ familiar, so I think I am missing something.

Reply:
A good start would be to report the build failure without this change as an issue and then we can see how upstream wants to fix it.

Please add a link to the issue above this `inreplace` so we can remove it when fixed.

Reply:
OK, I reported the issue and linked it to the patch.